view-app/src/index.js

```diff
@@ -21,7 +21,7 @@ const createWindow = () => {
     preload: path.join(__dirname, "open_dialog.js")}
     
   });
-
+  mainWindow.setMenu(null);
```

Comment:
This is already done by `mainWindow.setMenuBarVisibility(false);` (solution that is supposed to be better) five lines below. I did it here https://github.com/JakubBilski/tonations-recognition/pull/80, and you weren't seeing this bc this branch was not merged with master.

Reply:
done